### ISSUE TITLE
json formatting lost

### ISSUE THREAD

**Issue by vscaiceanu-1a** [ionic-team/trapeze#77]
Hello,
When updating the `capacitor.config.json` files for both iOS and Android I noticed that the json formatting is lost.
```yaml
vars:
  BUNDLE_ID:
    default: com.bundle.id
  PACKAGE_NAME:
    default:  com.bundle.id

platforms:
  ios:
    targets:
      App:
        bundleId: $BUNDLE_ID
        version: 16.4
        buildNumber: 198
        productName: Awesome App
        displayName: My Awesome App
        json:
          - file: App/App/capacitor.config.json
            set:
              appId: $BUNDLE_ID

  android:
    packageName: $PACKAGE_NAME
    versionName: 5.2.1
    versionCode: 197
    json:
      - file: app/src/main/assets/capacitor.config.json
        set:
          appId: $BUNDLE_ID
```

Is there an option that I am missing in order to reformat the json file after the update?

Thanks.

**Comment by mlynch** [comment]
Thanks for the issue, should be fixed in the next release

**Comment by mlynch** [comment]
Fixed in 3.0.6
